spec/features/user_delete_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'deleting a user' do
+  context 'with admin privilege' do
+    it 'deletes user' do
+      create :user, name: 'Adam'
+      admin = create :user, name: 'Bob', admin: true
+      login_as admin
+      visit admin_users_url
+      click_on 'Remove', match: :first
+      expect(page).to have_text 'User has been removed.'
+    end
+  end
+  context 'with master of ceremonies privilege' do
+    it 'will not delete a user' do
+      create :user, name: 'Adam'
```

Comment:
Right, the user needs to be present, sure. But their name shouldn't matter for your test. If it does, then you need a more specific way of running the test.